trulens_eval/examples/quickstart/snowpark_text2text_quickstart.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>Is 'It should look like this in' truncated? What does this mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>